src/interfaces/product.ts

```diff
@@ -0,0 +1,12 @@
+import type { Category } from './category';
+
+export interface Product {
```

Comment:
IProduct

Reply:
Link rules does not support such naming of interfaces.